doc/conf.py

```diff
@@ -54,6 +54,7 @@
     # "<source>": "<target>"
     "intro": "index.html",
     "support": "contact.html",
+    "ide-integration": "installation.html",
```

Comment:
I should have checked this earlier, but I don't think these actually work. The `RTD` output seems incorrect as well and the link:
https://pylint--6622.org.readthedocs.build/en/latest/user_guide/ide-integration.html
doesn't work.
https://pylint.pycqa.org/en/latest/user_guide/ide-integration.html

Reply:
¨The previous redirect worked, I'll fix this one.